src/channels/operations-channel.js

```diff
@@ -53,6 +102,16 @@ module.exports = function (options) {
             }
         },
 
+        /**
+         * Alert each subscriber about the operation and its parameters.
+         *
+         * **Example**
+         *
+         *      Flow.channel.operations.notify('myMethod', myMethodParam);
```

Comment:
This is the "response" of the operation usually, so `myMethodParam` = `myMethodResponse`